containers.go

```diff
@@ -78,3 +82,176 @@ func (c *Container) VNCWebSocket(vnc *VNC) (chan string, chan string, chan error
 
 	return c.client.VNCWebSocket(p, vnc)
 }
+
+func (c *Container) Feature(ctx context.Context) (hasFeature bool, err error) {
+	var feature struct {
+		HasFeature bool `json:"hasFeature"`
+	}
+	err = c.client.Get(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/feature", c.Node, c.VMID), &feature)
+	return feature.HasFeature, err
+}
+
+// This seems broken on the proxmox side: https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/lxc/{vmid}/interfaces
+// I wasn't able to make it work with the API. Tested on {"release":"7.3","repoid":"c3928077","version":"7.3-3"}
+// func (c *Container) Interfaces(ctx context.Context) (interfaces []string, err error) {
+// err = c.client.Get(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/interfaces", c.Node, c.VMID), &interfaces)
+// return interfaces, err
+// }
+
+func (c *Container) Migrate(ctx context.Context, params *ContainerMigrateOptions) (task *Task, err error) {
+	var upid UPID
+	if err := c.client.Post(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/migrate", c.Node, c.VMID), params, &upid); err != nil {
+		return nil, err
+	}
+	return NewTask(upid, c.client), nil
+}
+
+func (c *Container) Resize(ctx context.Context, disk, size string) (task *Task, err error) {
+	var upid UPID
+	if err := c.client.Post(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/resize", c.Node, c.VMID), map[string]interface{}{"disk": disk, "size": size}, &upid); err != nil {
+		return nil, err
+	}
+	return NewTask(upid, c.client), nil
+}
+
+func (c *Container) MoveVolume(ctx context.Context, params *VirtualMachineMoveDiskOptions) (task *Task, err error) {
+	var upid UPID
+	if err := c.client.Post(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/move_volume", c.Node, c.VMID), params, &upid); err != nil {
+		return nil, err
+	}
+	return NewTask(upid, c.client), nil
+}
+
+func (c *Container) RRDData(ctx context.Context, timeframe Timeframe, consolidationFunction ConsolidationFunction) (rrddata []*RRDData, err error) {
```

Comment:
any chance you learned anything writing the container RRDData func to help fix the virtualmachine func for this guy https://github.com/luthermonson/go-proxmox/issues/112 ?

Reply:
Well, I didn't realize the issue before so I just followed the same method for Containers as we do on VirtualMachines.  If you think that can cause issues we can remove or I can try to follow up on the issue and if I find anything I can come back to this one.

Reply:
i have no context, i thought it worked tbh and havent looked into it